purchases/sql/get_recurring_purchases.sql

```diff
@@ -1,11 +1,11 @@
 /*
 Name: get_recurring_purchases.sql
-Description: Get a user's purchased items where count > 2
-Usage: [user_id]
-Return: {name}
+Description: Get a user's purchased items where count >= 2
+Usage: [group_id]
+Return: {name, category, total cost}
 */
 
-SELECT p.name
+SELECT p.name, p.category, p.total_cost
```

Comment:
this assumes that the total cost of a recurring purchase is always the same...should we instead have this be a sum of the amounts of all of them, or do we purposely only want the same amount?